docs/reference/cli/pixi.md

```diff
@@ -0,0 +1,21 @@
+--8<-- [start:description]
+## Description
+
+The `pixi` command is the main entry point for the Pixi CLI.
+
+--8<-- [end:description]
+
+
+--8<-- [start:example]
+## Examples
```

Comment:
@ruben-arts don't you think the examples could live in the code directly?

Reply:
Yeah it could for sure. Need to figure out what we want visible in the `--help` vs the documentation pages.

Reply:
Do you mean that there should be no difference between `--help` and documentation, or that the `--help` should contain nothing?

Reply:
We've moved most of the example out of the code and into an extender file. This is done as the markdown example just don't look great in CLI and vice versa. If you know a better solution, let us know.